pkg/arvo/sys/hoon.hoon

```diff
@@ -13581,7 +13585,24 @@
             ;~  pfix  tar
               (stag %kthp (stag [%base %noun] ;~(pfix tec wide)))
             ==
-            (stag %kthp ;~(plug wyde ;~(pfix tec wide)))
+            ::
+            %+  stag
+              %kthp
+            ;~  plug
+              %+  sear
```

Comment:
This does feel awkward to me, and it's hard for me to feel confident that it's correct.  I wonder how the value-mode parser distinguishes between ``` ``@`foo ``` and ``` `@`foo  ```.  Presumably the same technique could be used here.

Reply:
`++scat` is most of the value mode parser, which is where this searing takes place. The way I understand the parser is the following:
```
we see a tec (backtick)
  handle pfix vat (@)
  handle pfix tar (*)
  handle plug type parser (wyde) with pfix tec value parser (wide) <- this does not work if wyde can also consume a tec (backtick), since we get an AST with [%bscl [%base %null] hoon] instead of [[%rock %n ~] hoon], so we sear
  handle pfix lus (+)
  else, just add a [%rock %n ~] to whatever hoon we are dealing with
```